### PR TITLE
remove chown

### PR DESCRIPTION
**Pull request by hillalex**
I ran this locally in the context of a deployment and it worked

### CODE REVIEW

File: bin/bb8/docker_rsync.py [hunk 1]
```diff
@@ -28,7 +28,6 @@ def _run_rsync(self, volumes, from_path, to_path, relative):
                # to the default rsh)
                "-e", ssh_cmd,
                "--perms", "--owner", "--group",
```
Comment (richfitz) [inline bin/bb8/docker_rsync.py]: Can we also drop the owner and group flags too?

**Review by richfitz — CHANGES_REQUESTED** [review]

File: bin/bb8/docker_rsync.py (hunk 1 above)
Comment (hillalex) [reply bin/bb8/docker_rsync.py]: Wait, do we even need the `--perms` flag in the end?

Comment (hillalex) [reply bin/bb8/docker_rsync.py]: oh yeah nevermind, I'm getting mixed up

**Review by MartinEden — APPROVED** [review]
So this is all tested and working fine? :)

**Comment by richfitz** [comment]
I have not had a chance to test this with the further flag removal

**Comment by hillalex** [comment]
I tested without the `chown`, just trying to re-run without the `user` and `group` flags but it's taking forever on my home network... @MartinEden want to try from the Imperial network?

**Review by richfitz — APPROVED** [review]

**Pull request merged**